test/sql/local/irc/test_lakekeeper_tpch.test

```diff
@@ -0,0 +1,55 @@
+# name: test/sql/local/irc/test_lakekeeper_tpch.test
+# description: test integration with iceberg catalog read
+# group: [irc]
+
+require avro
+
+require parquet
+
+require httpfs
+
+require iceberg
+
+require aws
+
+require tpch
+
+require-env LAKEKEEPER_SERVER_AVAILABLE
+
+statement ok
+create secret my_secret (
+	TYPE s3,
+	KEY_ID 'spark',
+	SECRET '2OR3eRvYfSZzzZ16MlPd95jhLnOaLM52'
+)
+
+statement ok
+attach 'demo' as my_datalake (
```

Comment:
We also support the `OAUTH2_SCOPE` and `OAUTH2_SERVER_URI` in the `create secret` statement right? 
Can we also get a test where those values are initialized there?

Reply:
I don't think we do currently, that's part of #142